# This message is used in ROSMovableKitTrayPlugin.cc
# This message is used in TaskManagerPlugin.cc

# Name of the movable tray, e.g., movable_tray_dark_wood_1
string movable_tray_name
# Type of the movable tray, e.g., movable_tray_dark_wood
string movable_tray_type
# Pose of the movable tray in the kit tray (on the agv)
geometry_msgs/Pose movable_tray_pose